### PR TITLE
Math Magicians: setup

### PR DESCRIPTION
**Pull request by ridwanediallo**
# Project requirements:

:one: The easiest way to create a React application is using Create React App (CRA) and following the instructions in this [guide](https://reactjs.org/docs/create-a-new-react-app.html#create-react-app).
:two: The application should run in the browser without errors.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,61 @@
...
+- [✔️] Milestone 1: **Project setup**
+- [✔️] Milestone 2: **Components**
+- [✔️] Milestone 3: **Events**
+- [x] Milestone 4: **Full website**
+- [ ] Milestone 5: **Tests**
+- [ ] Milestone 6: **Deploy**
+
+## sneak peek
+
+![Application screenshot](src/calculator.png)
```
Comment (zieeco) [inline README.md]: **[REQUIRED CHANGES]**

- Please, kindly remove all these sections as this is not needed in regards to following and maintaining the standard in the [microverse readme template](https://github.com/microverseinc/readme-template/blob/master/README.md) structure to make your readme consistent and professional.

File: README.md [hunk 2]
```diff
@@ -1 +1,61 @@
...
+To get a local copy up and running follow these simple exampl
+
+#### Live demo
+
+[Math Magicians Website](https://ridwanediallo.github.io/Math-magicians/) _Coming Soon_
+
+### **Option 2**
+
+#### Set up your own copy locally
+
```
Comment (zieeco) [inline README.md]: **[REQUIRED CHANGES]**

- Please, kindly remove the `Option 2` header so as to be consistent in following the format used in the [microverse readme template](https://github.com/microverseinc/readme-template/blob/master/README.md) structure. Instead, you can just leave the `Setup` header to make your readme consistent and professional.

File: README.md [hunk 3]
```diff
@@ -1 +1,61 @@
...
+- Clone [the GitHub Repository](https://github.com/ridwanediallo/Math-magicians.git)
+- Go to the Project `cd "Math-magicians`
+- Run `npm install` to install the _dependencies_
+- Run `npm start` to run the live server.
+- Run `npm test` to run the tests.
+
+## About me
+
```
Comment (zieeco) [inline README.md]: **[REQUIRED CHANGES]**

- Please, let's try and be consistent following the [microverse readme template](https://github.com/microverseinc/readme-template/blob/master/README.md) structure by using the "Authors" or "Author" header to appear more professional.

File: README.md [hunk 4]
```diff
@@ -1 +1,61 @@
...
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## 📝 License
```
Comment (zieeco) [inline README.md:48]: **[REQUIRED CHANGES]**

- Please, kindly add the "Acknowledgment" header before the `License` just after the `Show your support to be consistent following the [microverse readme template structure](https://github.com/microverseinc/readme-template/blob/master/README.md) to make your readme look more professional.

File: README.md [hunk 5]
```diff
@@ -1 +1,61 @@
...
+
+This project is under the [MIT](./LICENSE) license.
```
Comment (zieeco) [inline README.md]: **[REQUIRED CHANGES]**

- Please, kindly update the MIT license link as right now, the link is broken. You can use the MIT from the [microverse readme template](https://github.com/microverseinc/readme-template/blob/master/README.md) and add it to the root directory of your project to make your readme appear more professional.

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi, @ridwanediallo, 



Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- No Linter error :heavy_check_mark: 
- Project setup correctly :+1: 
- Descriptive PR :+1: 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by elisha2kyakpo1 — APPROVED** [review]
Hi 👋 @ridwanediallo, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- You did well to make the changes requested by the previous reviewer ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**